torch/_decomp/decompositions.py

```diff
@@ -5123,14 +5126,10 @@ def bernoulli(
 def isin_default(elements, test_elements, *, invert=False):
     if elements.numel() == 0:
         return torch.empty_like(elements, dtype=torch.bool)
-
     x = elements.view(*elements.shape, *((1,) * test_elements.ndim))
-    if not invert:
-        cmp = x == test_elements
-    else:
-        cmp = x != test_elements
     dim = tuple(range(-1, -test_elements.ndim - 1, -1))
-    return cmp.any(dim=dim)
+    res = (x == test_elements).any(dim=dim)
+    return torch.where(res, not invert, invert)
```

Comment:
NIT: could be simplified to
```suggestion
    return ~res if invert else res
```

Reply:
Thanks for the suggestions! Changed.